app/src/main/kotlin/com/simplemobiletools/clock/adapters/TimerAdapter.kt

```diff
@@ -130,16 +132,20 @@ class TimerAdapter(
                             is TimerState.Finished -> EventBus.getDefault().post(TimerEvent.Start(timer.id!!, timer.seconds.secondsToMillis))
                         }
                     } else {
-                        PermissionRequiredDialog(activity, R.string.allow_notifications_reminders, { activity.openNotificationSettings() })
+                        PermissionRequiredDialog(
+                            activity,
+                            com.simplemobiletools.commons.R.string.allow_notifications_reminders,
+                            { activity.openNotificationSettings() })
                     }
                 }
             }
 
             val state = timer.state
             val resetPossible = state is TimerState.Running || state is TimerState.Paused || state is TimerState.Finished
-            timer_reset.beInvisibleIf(!resetPossible)
-            val drawableId = if (state is TimerState.Running) R.drawable.ic_pause_vector else R.drawable.ic_play_vector
-            timer_play_pause.setImageDrawable(simpleActivity.resources.getColoredDrawableWithColor(drawableId, textColor))
+            timerReset.beInvisibleIf(!resetPossible)
+            val drawableId =
+                if (state is TimerState.Running) com.simplemobiletools.commons.R.drawable.ic_pause_vector else com.simplemobiletools.commons.R.drawable.ic_play_vector
```

Comment:
split this with if/else brackets, "val drawableId = " on an own line looks weird